peachjam/auth.py

```diff
@@ -6,16 +6,22 @@
 from django.contrib.auth.models import User
 from django.contrib.auth.views import redirect_to_login
 
+from peachjam.models import pj_settings
+
 
 class SocialAccountAdapter(DefaultSocialAccountAdapter):
     """Adapter that permits logins only from specific domains."""
 
-    ALLOWED_DOMAINS = ["laws.africa", "africanlii.org"]
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.allowed_domains = [
+            s.strip() for s in pj_settings().allowed_login_domains.split(",")
+        ]
 
     def pre_social_login(self, request, sociallogin):
         domain = sociallogin.email_addresses[0].email.split("@", 1)[1]
 
-        if domain not in self.ALLOWED_DOMAINS:
+        if domain not in self.allowed_domains:
```

Comment:
we should skip this check if no domains are configured

Reply:
I have set some default values `laws.africa` and `africanlii.org` so this is always present

Reply:
What if we want to allow logins for any domain?